va/va_enc_hevc.h

```diff
@@ -521,8 +529,12 @@ typedef struct _VAEncPictureParameterBufferHEVC {
         uint32_t        value;
     } pic_fields;
 
+    /** \brief if HierchLevelPlus1 > 0, it indicate  which layer current frame belong to */
+    uint8_t     HierchLevelPlus1;
+    /** \brief Reserved bytes for future use, must be zero */
+    uint8_t     va_byte_reserved[3];
     /** \brief Reserved bytes for future use, must be zero */
-    uint32_t                va_reserved[VA_PADDING_HIGH];
+    uint32_t                va_reserved[VA_PADDING_HIGH - 1];
 } VAEncPictureParameterBufferHEVC;
 
 /**
```

Comment:
we need to put comments in current num of B In Gop structure, to say it is to deprecate.